yellowbrick/gridsearch/pcolor.py

```diff
@@ -0,0 +1,225 @@
+
+    # Fit if necessary
+    if X is not None:
+        visualizer.fit(X, y)
+    else:
+        visualizer.draw()
+
+    # Return the axes object on the visualizer
+    return visualizer.ax
+
+
+class GridSearchColorPlot(GridSearchVisualizer):
+    """
+    Create a color plot showing the best grid search scores across two
+    parameters.
+
+    Parameters
+    ----------
+    model : Scikit-Learn grid search object
+        Should be an instance of GridSearchCV. If not, an exception is raised.
+
+    param_1 : string
+        The name of the parameter to be visualized on the horizontal axis.
+
+    param_2 : string
+        The name of the parameter to be visualized on the vertical axis.
+
+    ax : matplotlib Axes, default: None
+        The axes to plot the figure on. If None is passed in the current axes
+        will be used (or generated if required).
+
+    colormap : string or cmap, default: 'RdBu_r'
+        optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    Examples
+    --------
+    >>> from yellowbrick.gridsearch import GridSearchColorPlot
+    >>> from sklearn.model_selection import GridSearchCV
+    >>> from sklearn.svm import SVC
+    >>> gridsearch = GridSearchCV(SVC(),
+                                  {'kernel': ['rbf', 'linear'], 'C': [1, 10]})
+    >>> model = GridSearchColorPlot(gridsearch, param_1='kernel', param_2='C')
+    >>> model.fit(X)
+    >>> model.poof()
+    """
+
+    def __init__(self, model, param_1, param_2, colormap='RdBu_r', ax=None,
+                 **kwargs):
+        # super(GridSearchColorPlot, self).__init__(model, ax=ax, **kwargs)
+        super().__init__(model, ax=ax, **kwargs)
+        self.param_1 = param_1
+        self.param_2 = param_2
+        self.colormap = colormap
+
+    def draw(self):
+        # Project the grid search results to 2 dimensions
+        x_vals, y_vals, best_scores = param_projection(
+            self.cv_results_, self.param_1, self.param_2
+        )
+
+        # Mask nans so that they can be filled with a hatch
+        data = np.ma.masked_invalid(best_scores)
+
+        # Plot and fill in hatch for nans
+        mesh = self.ax.pcolor(data, cmap=self.colormap,
+                              vmin=np.nanmin(data), vmax=np.nanmax(data))
+        self.ax.patch.set(hatch='x', edgecolor='black')
+
+        # Ticks and tick labels
+        self.ax.set_xticks(np.arange(len(x_vals)) + 0.5)
+        self.ax.set_yticks(np.arange(len(y_vals)) + 0.5)
+        self.ax.set_xticklabels(x_vals, rotation=45)
+        self.ax.set_yticklabels(y_vals, rotation=45)
+
+        # Add the colorbar
+        cb = self.ax.figure.colorbar(mesh, None, self.ax)
```

Comment:
This is a running debate that I hope you might weigh in on: should color bars have the range (min, max) or should they have the range (0, 1)? Should this be an option we supply to the user?

Arguments for (min, max):

- they create greater color disparity so it's easier to tell values apart 
- They bound the area you're looking in 

Arguments for (0,1):

- You can easily compare multiple heatmaps with each other 
- The closer min is to max, the more similar they appear in this as opposed to the other.

Reply:
@ndanielsen @rebeccabilbro -- just wanted to highlight this discussion for you again here.

Reply:
In this case I'd advocate for min, max.  With grid search, you can use any of a number of error metrics, which may or may not be bounded by 1.  Furthermore when fine tuning parameters you may be optimizing over the 3rd or 4th decimal of the error metric, so fixing the range on a larger scale would make the visualization not very useful.